### PR TITLE
Feat:Home page added

### PR DESCRIPTION
**Pull request by ArpitPatni**
please review my home page of ecommerce website.

### CODE REVIEW

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
UI looks clean.
Suggestion:-  use comments inside html file to describe each section (not mandatory)

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+@import url(Styles/login.css);
+@import url(Styles/mycart.css);
+
+.right-nav{
+    margin-right: 50px;
+}
+.background-color{
+    background-color: #2874f0;
+}
```
Comment (vikrantkode) [inline styles.css]: use variables to define colors

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+.nav-heading{
+    color: white;
+}
+
```
Comment (vikrantkode) [inline styles.css]: same here

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+.container-body{
+    margin-top: 50px;
+    display: flex;
+    justify-content: space-around;
+}
+.container-part{
+    height: 120px;
+    width: 300px;
+    position: relative;
+    box-shadow: 1px 1px 10px black;
```
Comment (vikrantkode) [inline styles.css:28]: same here also

**Review by vikrantkode — CHANGES_REQUESTED** [review]
image

align items in the center in the nav bar
-change the suggested

File: styles.css (hunk 1 above)
Comment (ArpitPatni) [reply styles.css]: Thank you I have made all the corrections.

**Pull request merged**